src/index.ts

```diff
@@ -91,36 +93,32 @@ module.exports = (options?: any) => {
         process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
     }
 
-    ApplicationSupport.parseApplicationTags();
-
-    const logStreamName = Utils.getEnvVar(EnvVariableKeys.AWS_LAMBDA_LOG_STREAM_NAME);
-    const region = Utils.getEnvVar(EnvVariableKeys.AWS_REGION);
-    const functionVersion = Utils.getEnvVar(EnvVariableKeys.AWS_LAMBDA_FUNCTION_VERSION);
-    const applicationInstanceId = logStreamName ? logStreamName.split(']').pop() : Utils.generateId();
-
-    const pluginContext: PluginContext = new PluginContext({
-        applicationInstanceId,
-        applicationRegion: region ? region : '',
-        applicationVersion: functionVersion ? functionVersion : '',
-        requestCount: 0,
-        apiKey: config.apiKey,
-        timeoutMargin: config.timeoutMargin,
-        transactionId: null,
-        config,
-    });
-
-    const increaseRequestCount = () => pluginContext.requestCount += 1;
-
     return (originalFunc: any) => {
         // Check if already wrapped
         if (get(originalFunc, 'thundraWrapped', false)) {
             return originalFunc;
         }
 
+        ApplicationManager.setApplicationInfoProvider(new LambdaApplicationInfoProvider());
+        const applicationInfoProvider = ApplicationManager.getApplicationInfoProvider();
+        const applicationInfo = applicationInfoProvider.getApplicationInfo();
```

Comment:
We can directly use `const applicationInfo = ApplicationManager.getApplicationInfo()` here, instead of accessing it through the `applicationInfoProvider`